### ISSUE TITLE
50049 should not be assigned a personal tutor

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/uoj#9]
In the UDD, Caroline Bowman, student ID 50049 should not be assigned a personal tutor. We will have SSP configured to have Beth Kim as the default coach in SSP. Test case is to make sure student id 50049 who came over in the UDD is assigned to the default coach, Beth Kim.

**Comment by michaelwebbjisc** [comment]
STAFF_TUTOR_ID set to ""

**Comment by amason** [comment]
@michaelwebbjisc  in looking at student id 50049 in blue UDD data, I see this student id is set to "STAFF_ID":"5000","FIRST_NAME":"Beth","LAST_NAME":"Kim".

For my test case, I need student ID set to null; meaning, I want this student to come over with a personal tutor not assigned to I can test SSP's behavior.

Can you check your data for me again? SSP only uses current data not historical. We're using the stable api.

**Comment by michaelwebbjisc** [comment]
Can you try now - looks like I did exactly what I posted in the comment, which was wrong! (should have been TUTOR_STAFF_ID not STAFF_TUTOR_ID!

**Comment by amason** [comment]
@michaelwebbjisc the test case for student id 50049 passes. Let me look at each of the other test case scenarios and see how it affects what you mentioned above. I'll respond back here shortly.

**Comment by amason** [comment]
@michaelwebbjisc I need to check these requirements:

"Last Activity
Within a week, indicator should appear green in UI
Within 2 weeks, indicator should appear amber in UI
Over 2 weeks, indicator should appear red in UI

Calculation Requirements
Last activity - this is time since last xAPI activity in days."

Is that still accurate because Paul Spaude advised he thought the requirements changed to gray for scenarios. I have no record of that requirement change so can you advise if you wish to see the color based on the specifics above OR do you just want that field gray regardless of the last activity time?

**Comment by michaelwebbjisc** [comment]
Hi @amason - we've got a seperate repo for SSP:  We've not used it yet which is why you don't know about it! I've added you and paul as collaborators:

https://github.com/jiscdev/ssp-lite/issues

So can we use this issue log for UoJ data issues, and the SSP ones for SSP specific issues?

**Comment by amason** [comment]
@michaelwebbjisc  We see the issue now and it's on our end with Last Activity Time and Last Graded Work Date. 
Here are the requirements you gave us:

"3. Last Activity
Within a week, indicator should appear green in UI
Within 2 weeks, indicator should appear amber in UI
Over 2 weeks, indicator should appear red in UI

6. Date of Last Graded Work
Only text should display reflecting the Date of Last Graded Work. It should still behave as an alert, but do not color code this risk indicator in the UI
The value of the Date of Last Graded Work will be changing often based on when the faculty last graded work."

In SSP, we coded it backwards. We coded Last Activity Time to be a Date and Time stamp and Last Graded Work Date to be a number. We need to flip that in SSP so that Last Activity Time is in days and Last Graded Work Date is Date/time stamp.

So - let us get this code fixed and I'll then send you a new test scenario. Thanks and sorry for the confusion on this!
